wordpress-mobile/WordPress-Android
# Add Prologue

Add Prologue screen to the blogging reminders flow when coming from Site Settings too.

Fixes #14900 and #14911

NOTE:  Please ignore the copy, will be updated later

<img width=320 src=https://user-images.githubusercontent.com/990349/123035880-7a93f380-d42f-11eb-8b2a-ece88fe6851a.png>


To test:


- Turn on the blogging reminders flag (Me/App settings/Test feature configuration - BloggingRemindersFeatureConfig)
- Notice the blogging goals option appears (My Site > Settings > Blogging goals)
- Click on for e.g. "Twice a week" button under Blogging goals setting
- Notice the blogging reminders prologue screen is shown
- Click on "Set goals"
- Select days screen is shown, select days, then "Notify me" button is enabled
- Click on "Notify me" button
- Notice the epilogue screen is shown. Done button is enabled, Clicking Done button dismisses bottomsheet


## Regression Notes
1. Potential unintended areas of impact
Site Settings

2. What I did to test those areas of impact (or what existing automated tests I relied on)
NA

3. What automated tests I added (or what prevented me from doing so)
Update unit test

PR submission checklist:
